cmd/alerts/add_silence.go

```diff
@@ -1,104 +1,45 @@
 package alerts
 
 import (
-	"context"
 	"fmt"
 	"log"
-	"os"
-
 	"github.com/spf13/cobra"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/remotecommand"
-	"k8s.io/kubectl/pkg/scheme"
 )
 
 func NewCmdAddSilence() *cobra.Command {
 	return &cobra.Command{
-		Use:               "add-silence <cluster-id>",
+		Use:               "add-silence <cluster-id> <alertname> <comment>",
 		Short:             "add a new silence",
 		Long:              `add new silence`,
-		Args:              cobra.ExactArgs(1),
+		Args:              cobra.ExactArgs(3),
 		DisableAutoGenTag: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			AddSilence(args[0])
+			AddSilence(args[0],args[1],args[2])
 		},
 	}
 }
 
-var (
-	alertName = "alert01"
-	comment = "some info"
-)
-
-//osdctl alerts add-silence ${CLUSTERID} 
-func AddSilence(clusterID string){
-
-	_, kubeconfig, clientset, err := GetKubeCli(clusterID)
+//osdctl alerts add-silence ${CLUSTERID} ${alertname} ${comment}
+func AddSilence(clusterID string, alertName string, comment string){
```

Comment:
@bergmannf alertName would be provided by use during running command `add-silence` 

https://github.com/openshift/osdctl/pull/500#discussion_r1458598326

Reply:
How would the user then get a silence for all alerts?
This is the default way the alerts script was used, so this new way should allow this as well.